LeagueInformer/LeagueInformer/Program.cs

```diff
@@ -14,46 +14,55 @@ public class Program
 
         public static void Main(string[] args)
         {
-            if (ConnectionService.HasInternetConnection())
+            while (true)
             {
```

Comment:
Robienie nieskończonej pętli to nie jest dobry nawyk, postaraj się oprzeć to np. o option np. while(option != "4") lub option oprzeć o Integera i while(option != 4)